src/.clang-format

```diff
@@ -25,53 +25,74 @@ IndentWidth: 4
 UseTab: Never
 TabWidth: 8
 AccessModifierOffset: -4
-ConstructorInitializerIndentWidth: 4
-ContinuationIndentWidth: 4
+ConstructorInitializerIndentWidth: 0
```

Comment:
Yeah, I spotted them being dragged to the right when we applied the "existing" file to `XMLimport` class a few weeks ago, and I was _requested_ to _refrain_ from _adjusting_ the previous settings in this file in https://github.com/Mudlet/Mudlet/pull/372. \**looks across to another member of the team*\* :wink: